### ISSUE TITLE
Many commit messages are non-descriptive.

### ISSUE THREAD

**Issue by JesperWe** [agentejo/cockpit#565]
This is just a small request on my part: It is hard to follow along with what  is happening when there are many commits saying "Minor improvements" and similar. 

Please consider writing more descriptive commit messages to allow folks like me who are watching your commits to understand better what was actually changed!

**Comment by aheinze** [comment]
Hi Jesper,

I understand your issue, but most of the times it is really just minor improvements that aren't really noticeable by the user sitting in front of cockpit. But I'll try to improve on the that next time ;-)

Greets,
Artur
